application/teams/__init__.py

```diff
@@ -1,8 +1,10 @@
 """Provides Team functionality and routes"""
 from flask import Blueprint
 
-from .teams_view import TeamView, TeamsView
+from .teams_view import TeamView, TeamsView, MembersView, AllTeamsView
 
 TEAMS = Blueprint('teams', __name__)
 TEAMS.add_url_rule('', view_func=TeamsView.as_view('teamsview'))
 TEAMS.add_url_rule('/<id>', view_func=TeamView.as_view('teamview'))
+TEAMS.add_url_rule('/<id>/members', view_func=MembersView.as_view('membersview'))
```

Comment:
Adding a member via POST is intuitively REST. But since you need to know the team_id to perform this action (which we don't have, we only have the join_key), it is not exactly what we need.